mava/utils/logger.py

```diff
@@ -295,11 +330,7 @@ def _make_multi_logger(cfg: DictConfig) -> BaseLogger:
 
 def get_logger_path(config: DictConfig, logger_type: str) -> str:
     """Helper function to create the experiment path."""
-    return (
-        f"{logger_type}/{config.logger.system_name}/{config.env.env_name}/"
-        + f"{config.env.scenario.task_name}"
-        + f"/envs_{config.arch.num_envs}/seed_{config.system.seed}"
-    )
+    return f"{logger_type}/{config.logger.system_name}"
```

Comment:
I feel like removing all these things will make it very likely to always have a long json file or will it create a new one each time?

Reply:
In the `JsonLogger` class init we add a unique token to the json file name so it should always be unique. I just simplified this because it was a relic of old Mava experiments we did when we were still using sacred.